c/loop-simple/README.txt

```diff
@@ -1,2 +1,3 @@
 Simple loop example programs.
- - deep-nested: Intended to show that execution-based witness violators are not always a good solution. Written by Philipp Berger, RWTH Aachen University.
\ No newline at end of file
+ - deep-nested: Intended to show that execution-based witness violators are not always a good solution. Written by Philipp Berger, RWTH Aachen University.
```

Comment:
**suggestion**: maybe fix the typo here when you are at it? (violators should be validators)